### ISSUE TITLE
"vagrant up" broken

### ISSUE THREAD

**Issue by bricas** [IQSS/dataverse#2300]
I tried to "vagrant up" the v4.0.1 tag and it failed with the "can't find .war file" error.

I had a look at the installed and came up with the following diff to make the installer work:

```patch
diff --git a/scripts/installer/install b/scripts/installer/install
index 051c4d1..069916a 100755
--- a/scripts/installer/install
+++ b/scripts/installer/install
@@ -317,14 +317,14 @@ my $WARFILE_LOCATION = "dataverse.war";

 unless ( -f $WARFILE_LOCATION ) {
     my $DATAVERSE_VERSION = "";
-    my $DATAVERSE_VERSION_FILE = "../../src/java/VersionNumber.properties";
+    my $DATAVERSE_VERSION_FILE = "../../src/main/java/VersionNumber.properties";
     if ( -f $DATAVERSE_VERSION_FILE )
     {
        open DVF, $DATAVERSE_VERSION_FILE;
        $DATAVERSE_VERSION = <DVF>;
        close DVF;
        $DATAVERSE_VERSION=~s/[ \n\t\r]*$//;
-       $DATAVERSE_VERSION=~s/^dataverse.version=//;
+       $DATAVERSE_VERSION=~s/^version.number=//;

        $WARFILE_LOCATION = "../../target/dataverse-" . $DATAVERSE_VERSION . ".war";
     }
```

**Comment by pdurbin** [comment]
It's broken on master too:

```
...
==> standalone: WARNING: Can't find the project .war file!
==> standalone: 	Are you running the installer in the right directory?
==> standalone: 	Have you built the war file?
==> standalone: 	(if not, build the project and run the installer again)
murphy:dataverse pdurbin$ 
```

Thanks for reporting this @bricas! I'll probably try to fix this in the 4.0.2 branch.

**Comment by pdurbin** [comment]
Ok, the good news is that `vagrant up` works just fine on the 4.0.2 branch. It builds and deploys the war file and and end I can see the running app at http://localhost:8888/

So it's a bummer that `vagrant up` doesn't work from the v4.0.1 tag (should we note this somewhere) but as soon as we merge 4.0.2 into master it will work there again, which is more important to me.

I guess I'll leave this issue open until I've merged 4.0.2 to master. @bricas I'm happy to hear any more thoughts though!

**Comment by bricas** [comment]
Ah yes, i see the fix in 436a39c52bf15d65b2f23f8e646b3259d0b3258b.

It doesn't much matter do me so long as the latest stable is functional.

I see 33 open issues (candidates) for 4.0.2, which seems like it may be a ways off. This seems like a pretty critical issue to me as it's one of the simplest ways to install and test out the system -- if there was room to push a 4.0.1-1 (or whatever) that would be great, otherwise I'm happy just running 4.0.2 "dev" since we're actually just evaluating the system at this point.

Since it's already fixed, I'm happy to leave the rest up to y'all :)

**Comment by pdurbin** [comment]
> I see 33 open issues (candidates) for 4.0.2

And another 51 open in 4.0.2 (35 of which are not yet in QA) **but** we're supposed to hit code freeze on Monday so perhaps the end of 4.0.2 is near:

![milestones_-_iqss_dataverse_-_2015-06-30_21 38 22](https://cloud.githubusercontent.com/assets/21006/8445855/933a06c4-1f70-11e5-8f7e-5f33423739d5.png)

Anyway, at the moment 4.0.2 is stable. I'm glad you're finding the Vagrant environment useful. Please get in touch if you have any questions during your evaluation!

**Comment by bencomp** [comment]
This is related to #2165 (and my PR #2259): if the version number in pom.xml doesn't match the one in VersionNumber.properties, the installer script cannot find the WAR, even if it builds it itself.

**Comment by pdurbin** [comment]
I just verified that even after renaming 4.0.2 to 4.1 in #2348 `vagrant up` still works in the 4.0.2 branch (we didn't rename the branch, and probably won't bother). Passing to QA.

**Comment by pdurbin** [comment]
I just merged the 4.1 branch (4.0.2 technically since we renamed it in #2348 ) into master and `vagrant up` still works on master. I'd say we can close this ticket.

**Comment by kcondon** [comment]
Closing based on Phil's testing
